posthog-core/test/posthog.flush.spec.ts

```diff
@@ -1,5 +1,7 @@
 import { createTestClient, PostHogCoreTestClient, PostHogCoreTestClientMocks } from './test-utils/PostHogCoreTestClient'
-import { waitForPromises } from './test-utils/test-utils'
+import { delay, waitForPromises } from './test-utils/test-utils'
+import { PostHogPersistedProperty } from '../src'
+import useRealTimers = jest.useRealTimers
```

Comment:
Is this valid TS?

Reply:
Oh I think it is but I don't like it, I think pycharm probably  autogenerated it, I'll remove it